### ISSUE TITLE
Latest version of Inkscape cannot open 

### ISSUE THREAD

**Issue by sawyerbfuller** [upb-lea/Inkscape_electric_Symbols#8]
**Describe the bug**
.svg drawing fails to load in latest version of Inkscape: 1.2.2 (b0a84865, 2022-12-01)

**Expected behavior**
Drawing loads

**Screenshots**
<img width="471" alt="image" src="https://user-images.githubusercontent.com/58706249/224813504-92cdb44f-a3f1-4dcf-bff3-d51501c33ca9.png">

**Desktop (please complete the following information):**
 - Mac OS Monterey, ARM64 (M1)

**Additional context**
Love this idea and hoping I can use it, thanks for providing!

**Comment by gituser789** [comment]
Hi there,
thanks for this hint.

I had a look at my inkscape version, which is exactly the same `Inkscape 1.2.2 (b0a8486541, 2022-12-01)`, running on linux system.

 * Can you open other Inkscape-drawings on your computer without any problems?
 * Does this also happen on your system with a lower version number of inkscape?

**Comment by sawyerbfuller** [comment]
Peculiar. 

I also encountered the exact same error message on:
* x86-64 version of inkscape 1.2.2
* M1/ARM64 version of inkscape 1.2.0

I went through and opened about 10 different drawings I have previously created and they all opened fine and none of them gave this error. 

Then I rest all preferences:  inkscape -> preferences -> system -> reset all preferences. Preferences reset, but still the same error. 

My best guess at this point is that it maybe it has to do with some settings I was playing around with when inkscape had a conflict with mac os Monterey (I don't remember what I did). Closing for now.
